opennms-doc/guide-admin/src/asciidoc/text/telemetryd/troubleshooting.adoc

```diff
@@ -0,0 +1,57 @@
+
+[[telemetryd-troubleshooting]]
+=== Troubleshooting
+
+Configuring _telemetryd_ involves a few components.
+This section gives a few troubleshooting hints to verify if the configuration and the network setup is correct.
+
+.Ensure the Java process is listening on a specific port
+[source]
+----
+ss -lnp sport = :8877
+
+Netid State  Recv-Q Send-Q Local Address:Port Peer Address:Port
+udp   UNCONN 0      0      *:8877                           *:* users:(("java",pid=585,fd=1226))
+----
+
+.Ensure Firewall is not blocking incoming traffic
+[source]
+----
+sudo iptables -L
+----
+
+.Example to set a firewall rule with firewalld for 8877/udp
+[source]
+----
+firewall-cmd --add-port=8877/udp --zone=public
+----
+or
+
+.Example to set a firewall rule with iptables for 8877/udp
+[source]
+----
+iptables -I INPUT -p udp -m udp --dport 8877 -j ACCEPT
+----
+
+.Check if packets can be written to the internal queue for the specific protocol `Netflow-5`
+[source]
+----
+tail -f ${OPENNMS_HOME}/logs/telemetryd.log | grep "JmsConsumer[OpenNMS.Sink.Telemetry.Netflow-5]"
```

Comment:
The error message referenced bellow, won't actually appear in this log file,it will appear on the Minion.

Reply:
Also, this branch does not currently have support for Netflow.

Reply:
Do you have a consumer which exist on 21.0.0 we can use as an example?

Reply:
This is a funny question, since you restructured the documentation for the existing adapters.